SimCalorimetry/HcalSimProducers/python/hcalSimParameters_cfi.py

```diff
@@ -120,10 +124,14 @@
 run2_HE_2017.toModify( hcalSimParameters,
     he = dict(
         photoelectronsToAnalog = cms.vdouble([57.5]*14),
-        pixels = cms.int32(27370), 
-        sipmDarkCurrentuA = cms.double(0.055),
-        sipmCrossTalk = cms.double(0.32),
         doSiPMSmearing = cms.bool(True),
+        sipmTau = cms.double(10.),
+    )
+)
+from Configuration.Eras.Modifier_run2_HCAL_2017_cff import run2_HCAL_2017
+run2_HCAL_2017.toModify( hcalSimParameters,
+    ho = dict(
+        doSiPMSmearing = cms.bool(True)
```

Comment:
HO did not change in 2016 vs 2017, right?
IMO, it makes more sense to have the same configuration for the same detector setup for consistency.